docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,129 @@
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+# Converting To WinUI3 Using CSharp Conversion Analyzers
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+The purpose of this project is to create a porting solution in the form of Code Analyzers/Fixes to allow Developers to convert WinUI2 projects to the new WinUI3 format. As Microsoft moves toward a purely public SDK, its Xaml UI components are being lifted out of the OS and deployed in WinUI3.
+
+- WinUI2 is an official library with support for native Windows UI elements for Windows apps. 
+- WinUI3 is the new offical UI framework for both Windows Desktop and UWP apps. 
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+This tool assists with the conversion process by providing Code Analyzers/Fixes to convert existing WinUI2 C# projects to WinUI3.
+
+### What It Does
+
+- Analyze/Convert C# files
```

Comment:
This isn't it's own bullet, is it? The rest of the bullets are below this?

Reply:
removed bullet